ttnn/cpp/ttnn/operations/conv/conv2d/device/multi_core_optimized_conv_sharded/optimized_conv_op_sharded_v2.cpp

```diff
@@ -2,17 +2,17 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
-#include "ttnn/tensor/tensor_utils.hpp"
-#include "ttnn/operations/experimental/auto_format/auto_format.hpp"
-#include "ttnn/operations/conv/conv2d/device/optimized_conv_op.hpp"
-#include "ttnn/deprecated/tt_dnn/op_library/sharding_utilities.hpp"
-#include "ttnn/operations/sliding_window/sliding_window.hpp"
-#include "tt_metal/common/work_split.hpp"
 #include "tt_metal/common/constants.hpp"
+#include "tt_metal/common/work_split.hpp"
 #include "tt_metal/detail/tt_metal.hpp"
 #include "tt_metal/detail/util.hpp"
 #include "tt_metal/host_api.hpp"
 #include "tt_stl/reflection.hpp"
+#include "ttnn/operations/conv/conv2d/device/optimized_conv_op.hpp"
+#include "ttnn/operations/experimental/auto_format/auto_format.hpp"
+#include "ttnn/operations/moreh/sharding_utilities.hpp"
+#include "ttnn/operations/sliding_window/sliding_window.hpp"
+#include "ttnn/tensor/tensor_utils.hpp"
```

Comment:
can you revert format chages, and include order so that we can focus on actual changes in this PR?

Reply:
This is for the whole PR

Reply:
Sure, I will try to done it before mid night ICT.
Actually I thought about it but ignore because of the pre-commit checks!

Reply:
@pavlejosipovic please take a look again, the clang-format is reverted.